RecoMuon/TrackerSeedGenerator/plugins/TSGForRoadSearch.cc

```diff
@@ -426,6 +426,7 @@ void TSGForRoadSearch::makeSeeds_4(const reco::Track &muon, std::vector<Trajecto
             edm::LogWarning(theCategory) << "ran out of layers to find a seed: no seed.";
             return;
           }
+          [[fallthrough]];
```

Comment:
This looks like a bug, as it should rather ```break``` here instead: @Fedespring @bmahakud

Reply:
Yes it is a bug, fallthrough has to be replaced with break.